test/validators/htmlValidator.js

```diff
@@ -45,7 +45,7 @@ describe('HTMLValidator', function () {
         output.should.include('missing-html.html');
       });
 
-      it('Should return an error string when there is an unavailable layout', async function () {
+      it('Should return an error string when there is an unavailable layout pointing to the layout file', async function () {
```

Comment:
What is meant with "an unavailable layout pointing to the layout file"?

Reply:
Before these changes, if template A extends layout B and B has invalid frontmatter or doesn’t exist. The error would show an error in A, now it shows an error in B. This tests makes sure it does.

Reply:
Ok, can you add that as a comment as well, because that is not clear from the title of test?